README.md

```diff
@@ -303,12 +305,27 @@ trigram-index = true
 # in memory. This determines how often it will check FS
 # for new or deleted metrics.
 scan-frequency = "5m0s"
+# Control trie index
+#  This index is built as an alternative to trigram index, with shorter indexing
+#  time and less memory usage (around 2 - 5 times). For most of the queries, trie
+#  is faster than trigram. For queries with keyword wrap around by widcards (like
+#  ns1.ns2.*keywork*.metric), trigram indexi performs better. Trie index could
+#  be speeded up by enabling adding trigrams to trie, at the cost of memory usage (
+#  depends on the facotr, the memory usage of trie would be at least doubled).
+trie-index = false
+trie-index-with-trigrams = false
+
 # Maximum amount of globs in a single metric in index
 # This value is used to speed-up /find requests with
 # a lot of globs, but will lead to increased memory consumption
 max-globs = 100
 # Fail if amount of globs more than max-globs
 fail-on-max-globs = false
+# Maximum metrics could be returned by glob/wildcard in find request
+max-metrics-globbed  = 30000
+# Maximum metrics could be returned in render request
```

Comment:
What will be returned in the failure scenario here?

Reply:
Now it just silently truncated the returning result to this upper limit for easy implementation. I could tweak it if you have a better idea. :D